packages/flutter/test/rendering/mouse_tracking_test.dart

```diff
@@ -16,8 +14,8 @@ import 'package:vector_math/vector_math_64.dart' show Matrix4;
 import '../flutter_test_alternative.dart';
 import './mouse_tracking_test_utils.dart';
 
-TestMouseTrackerFlutterBinding _binding = TestMouseTrackerFlutterBinding();
-MouseTracker get _mouseTracker => RendererBinding.instance.mouseTracker;
+TestMouseTrackerFlutterBinding? _binding = TestMouseTrackerFlutterBinding();
```

Comment:
Why nullable? Looks like it is never null?

Reply:
Same here, there is a dead null check on line 21 otherwise.